src/pages/common/_crsfconfig_page.c

```diff
@@ -0,0 +1,83 @@
+/*
+ This project is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ Deviation is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with Deviation.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#if HAS_CRSF_CONFIG
+
+#include "crsf.h"
+
+#ifndef EMULATOR
+crsf_device_t crsf_devices[CRSF_MAX_DEVICES];
+#else
+crsf_device_t crsf_devices[] = {
+{
+  .address = 238,
+  .number_of_params = 24,
+  .params_version = 1,
+  .serial_number = 18661,
+  .hardware_id = 65539,
+  .firmware_id = 600,
+  .name = "XF Transmitter",
+},
+{
+  .address = 236,
+  .number_of_params = 40,
+  .params_version = 1,
+  .serial_number = 29509,
+  .hardware_id = 73760,
+  .firmware_id = 600,
+  .name = "XF Nano RX",
+},
+};
+#endif
+
+static struct crsfconfig_page * const mp = &pagemem.u.crsfconfig_page;
+static struct crsfconfig_obj * const gui = &gui_objs.u.crsfconfig;
+static u16 current_selected = 0;
+
+static u32 last_update;
+static u8 number_of_devices;    // total known
+
```

Comment:
last_update and number_of_devices can probably also move to pagemem

Reply:
Only last_update, as number_of_devices is need for smooth "pop" to config page from device page.  Commit ee3c2ab4